sdks/python/apache_beam/dataframe/frames.py

```diff
@@ -367,9 +384,14 @@ def map_index(df):
       'astype', base=pd.core.generic.NDFrame)
   copy = frame_base._elementwise_method('copy', base=pd.core.generic.NDFrame)
 
+  @frame_base.with_docs_from(pd.DataFrame)
   @frame_base.args_to_kwargs(pd.DataFrame)
   @frame_base.populate_defaults(pd.DataFrame)
   def tz_localize(self, ambiguous, **kwargs):
+    """``ambiguous`` may not be set to 'infer' as it's semantics are
```

Comment:
"may not" -> "cannot"

As a general rule, "may" is for permission or an uncertain outcome ("You may find that passing in `foo` improves performance"), and "can" is for possibility. So if we prevent a user from doing something (i.e. it's not possible), we usually say, "You cannot do foo..."

Reply:
Ah thanks for the explanation, this makes sense. Fixed.